manual/de/ch_Finance-Quote.xml

```diff
@@ -307,33 +296,32 @@ SYNOPSIS
         </para>
       </abstract>
 
-      <formalpara>
-        <title>Beispiel mit defekten Abhängigkeit zu weiteren Softwarepaketen</title>
-
-        <para>
+      <informalexample><?dbfo pgwide="1"?>
```

Comment:
Another task:
suppress page breaks inside screens.

The PDF looks ugliy here.